packages/storybook/cxl-ui/cxl-featured-course-card/dashboard-slider.stories.js

```diff
@@ -29,11 +29,15 @@ const args3 = {
 
 const Template = () => html`
   <cxl-tabs-slider theme="minimal cxl-featured-course-slider">
-    <vaadin-tab theme="cxl-featured-course-slider"
+    <vaadin-tab disabled theme="cxl-featured-course-slider"
```

Comment:
Note that adding `disabled` for tabs here is necessary to workaround `cxl-tabs-slider` scrolling triggered by clicks on the cards. A definitive fix would require overriding a sizeable part of some logic in mixins used by `vaadin-tabs`.